### ISSUE TITLE
Analysis on current Prospects in Salesforce

### ISSUE THREAD

**Issue by nicholassewitz** [artsy/clouds#79]
https://docs.google.com/spreadsheets/d/1c9oB00eBjFK8B5PMmAMlf0qrFZW9ZZaoP33885MLsaU/edit?ts=59a6d619#gid=1952744508

**Comment by nicholassewitz** [comment]
alex.banfich [5:14 PM] 
Hi both, just shared a doc with you (greg I sent it to you yesterday) -- I did a little "audit" of prospect=True leads in salesforce


[5:14] 
I chose my team's 3 top sub-regions - US (New York), US (California), US (Southwest)


[5:15] 
and I picked 50 leads somewhat randomly (for NY I chose the top 50, Southwest I chose the bottom 50, and California I chose 50 in the middle of the list)


[5:15] 
It took me ~30 minutes to qualify/disqualify 50 leads at a time, and I found that on average, 48% of the leads in those lists were not qualified (edited)


nicholas [5:16 PM] 
so 25 qualified leads in 30 minutes


[5:16] 
that seems really good


alex.banfich [5:16 PM] 
yeah, if there was an existing opp, i assumed it was qualified


[5:16] 
if not, i looked at the website


[5:17] 
This came from Sydney telling me why she prefers to self-prospect over look in Salesforce


[5:18] 
and after going thru this exercise, I get it - I too would rather self-prospect high-quality, contemporary galleries not in Salesforce before spending my time disqualifying jewelry galleries (there are a lot of those, for example)


[5:18] 
so from here, my recommendation would be the following - A. I think we need to continue Bearden imports of galleries that are likely to be qualified (if we think more like the Artnet leads are in there?) and B. Need to go in and disqualify a lot of these


[5:19] 
I'm not sure if GPAs should do the disqualifying (not a good use of their time, will result in lower opp #s) or if myself/Saskia should do it...or if we do a tiering party one day where we all spend an hour doing it (all GPAs for example) and make it a thing


[5:19] 
let me know your thoughts!


nicholas [5:20 PM] 
really great excercise here


[5:20] 
another option is to have them limited to galleries


[5:20] 
and not design galleries


[5:20] 
i assume a lot of those tier 5's were design


alex.banfich [5:20 PM] 
they were


nicholas [5:21 PM] 
i bet that brings the average qualified up 10-20%


alex.banfich [5:21 PM] 
agreed ^


[5:21] 
it would


nicholas [5:21 PM] 
and possibly we do a design tiering party


[5:21] 
down the road


alex.banfich [5:21 PM] 
only other thought is our data - what we report to Sebastian on # of qualified leads in Salesforce - this has implications on that


[5:22] 
but ok, assume we take out design from Prospect=True for a quick fix, what about populating Salesforce with more leads?


nicholas [5:22 PM] 
well we have all of the leads


[5:22] 
that are mqls


alex.banfich [5:22 PM] 
How do we allow GPAs to prospect MQLs outside of Patrons lists? (edited)


[5:23] 
(this is assuming our  line of thinking that we need GPAs almost exclusively in Salesforce and not self-prospecting)


nicholas [5:24 PM] 
well i still think that 25 prospects in an half hour (edited)


[5:24] 
is better than them self prospecting


[5:24] 
so i dont think we should rule that off the table


[5:26] 
there are ~1800 gallery prospects in EU and ~1000 in NA


[5:26] 
thats still a lot plus their artnet


alex.banfich [5:26 PM] 
Fair. To play devil's advocate (and what GPAs would say to that) is that out of those 25, they would get 5 churned partners, 5-10 qualified furniture galleries, and then ~10 other galleries
